dash_table/Format.py

```diff
@@ -0,0 +1,270 @@
+import inspect
+import collections
+
+
+def get_named_tuple(name: str, dict: dict):
+    return collections.namedtuple(name, dict.keys())(*dict.values())
+
+
+Align = get_named_tuple('align', {
+    'default': '',
+    'left': '<',
+    'right': '>',
+    'center': '^',
+    "right_sign": '='
+})
+
+Group = get_named_tuple('group', {
+    'no': '',
+    'yes': ','
+})
+
+Padding = get_named_tuple('padding', {
+    'no': '',
+    'yes': '0'
+})
+
+Prefix = get_named_tuple('prefix', {
+    'yocto': 10**-24,
+    'zepto': 10**-21,
+    'atto': 10**-18,
+    'femto': 10**-15,
+    'pico': 10**-12,
+    'nano': 10**-9,
+    'micro': 10**-6,
+    'milli': 10**-3,
+    'none': None,
+    'kilo': 10**3,
+    'mega': 10**6,
+    'giga': 10**9,
+    'tera': 10**12,
+    'peta': 10**15,
+    'exa': 10**18,
+    'zetta': 10**21,
+    'yotta': 10**24
+})
+
+Scheme = get_named_tuple('scheme', {
+    'default': '',
+    'decimal': 'r',
+    'decimal_integer': 'd',
+    'decimal_or_exponent': 'g',
+    'decimal_si_prefix': 's',
+    'exponent': 'e',
+    'fixed': 'f',
+    'percentage': '%',
+    'percentage_rounded': 'p',
+    'binary': 'b',
+    'octal': 'o',
+    'lower_case_hex': 'x',
+    'upper_case_hex': 'X',
+    'unicode': 'c'
+})
+
+Sign = get_named_tuple('sign', {
+    'default': '',
+    'negative': '-',
+    'positive': '+',
+    'parantheses': '(',
+    'space': ' '
+})
+
+Symbol = get_named_tuple('symbol', {
+    'none': '',
+    'currency': '$',
+    'binary': '#b',
+    'octal': '#o',
+    'hex': '#x'
+})
+
+Trim = get_named_tuple('trim', {
+    'no': '',
+    'yes': '~'
+})
+
+
+class Format():
+    def __init__(self, **kwargs):
+        self.locale = {}
+        self.nully = ''
+        self.prefix = Prefix.none
+        self.specifier = {
+            'align': Align.default,
+            'fill': '',
+            'group': Group.no,
+            'width': '',
+            'padding': Padding.no,
+            'precision': '',
+            'sign': Sign.default,
+            'symbol': Symbol.none,
+            'trim': Trim.no,
+            'type': Scheme.default
+        }
+
+        valid_methods = [m for m in dir(self) if m != '__init__' and m != 'create' and inspect.ismethod(getattr(self, m))]
+
+        for arg in kwargs:
+            if arg not in valid_methods:
+                raise Exception('{0} is not a format method. Expected one of'.format(arg), str(list(valid_methods)))
+
+            getattr(self, arg)(kwargs[arg])
```

Comment:
Stumbled my way to this solution for applying the values to the underlying data structure in the instance

Reply:
Sure, that'll work. Style point, since `args` is typically used for positional arguments, I'd avoid `arg` as the name. Perhaps:
```py
for kw, val in kwargs.items():
    ...
    getattr(self, kw)(val)
```

Reply:
Agreed. Also, forgot about Python supporting iterating on two lists at a time!.. although this is destructuring a tuple, not two lists..